cli.py

```diff
@@ -51,38 +52,46 @@ def attr_upd():
         return
 
     backend_selector = BackendSelector(selected_brand, selected_region)
+    session = aiohttp.ClientSession()
 
-    auth = Auth(backend_selector, args.email, args.password)
+    auth = Auth(backend_selector, args.email, args.password, session)
     await auth.do_auth(store=False)
-    appliance_manager = AppliancesManager(backend_selector, auth)
+    appliance_manager = AppliancesManager(backend_selector, auth=auth, session=session)
```

Comment:
```suggestion
    appliance_manager = AppliancesManager(backend_selector, auth, session)
```
we are not missing any arguments, so we do not need to have keyword args here